fiat-rust/Cargo.toml

```
@@ -2,12 +2,13 @@
 name = "fiat-crypto"
 version = "0.2.10"
```

We should bump the version here, right?
```suggestion
version = "0.3.0"
```